packages/stark-testing/karma.conf.ci.js

```diff
@@ -43,7 +43,8 @@ const rawKarmaConfig = {
 			],
 			resolve: {},
 			transforms: [
-				require("karma-typescript-angular2-transform") // see https://github.com/monounity/karma-typescript-angular2-transform
+				require("karma-typescript-angular2-transform"), // see https://github.com/monounity/karma-typescript-angular2-transform
```

Comment:
Can you update the URL of the link, now it is https://github.com/monounity/karma-typescript/tree/master/packages/karma-typescript-angular2-transform